airflow/www/views.py

```diff
@@ -1034,6 +1035,30 @@ def _iter_parsed_moved_data_table_names():
                     "warning",
                 )
 
+        scarf_url = ""
+        if settings.IS_SCARF_ANALYTICS_ENABLED:
+            scarf_domain = "https://apacheairflow.gateway.scarf.sh"
+
+            python_version = platform.python_version()
+            platform_sys = platform.system()
+            platform_arch = platform.machine()
+            db_name = settings.engine.dialect.name
+            db_version = settings.engine.dialect.server_version_info
+            if db_version:
+                # Example: (1, 2, 3) -> "1.2.3"
+                db_version = ".".join(map(str, db_version))
+            executor = conf.get("core", "EXECUTOR")
+
+            # Path Format:
+            # /{version}/{python_version}/{platform}/{arch}/{database}/{db_version}/{executor}/{num_dags}
+            #
+            # This path redirects to a Pixel tracking URL
+            scarf_url = (
+                f"{scarf_domain}/webserver"
+                f"/{version}/{python_version}"
+                f"/{platform_sys}/{platform_arch}/{db_name}/{db_version}/{executor}/{all_dags_count}"
+            )
```

Comment:
There lines seem to be very redundant to the scheduler helper function in `airflow/utils/scarf.py` - I'd propose to make the utility generic and use it in both places to prevent redundant distributed logic.

Reply:
Updated